packages/libs/react-components/src/components/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import { SystemIcons } from './../../';
+import { INotificationProps, Notification } from '.';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  {
+    selectIcon: keyof typeof SystemIcons;
+    text: string;
+  } & INotificationProps
+> = {
+  title: 'Notification',
+  argTypes: {
+    selectIcon: {
+      options: Object.keys(SystemIcons) as (keyof typeof SystemIcons)[],
+      control: {
+        type: 'select',
+      },
+    },
+    title: {
+      control: {
+        type: 'text',
+      },
+    },
+    description: {
+      control: {
+        type: 'text',
+      },
+    },
+    simple: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    color: {
+      options: [
+        'default',
+        'primary',
+        'secondary',
+        'positive',
+        'warning',
+        'negative',
```

Comment:
I would recommend making the variant a separate object where the styled component is being created and importing it here to get the keys so that if those variants are ever changed, there is one source of truth and every dependent usage will be updated. I believe you can see an example of this in IconButton